graphql_compiler/__init__.py

```diff
@@ -202,11 +202,15 @@ def get_graphql_schema_from_orientdb_schema_data(schema_data, class_to_field_typ
 
 
 def graphql_to_redisgraph_cypher(schema, graphql_query, parameters, type_equivalence_hints=None):
-    """Compile the GraphQL input using the schema into a Cypher query and associated metadata.
+    """Compile the GraphQL input using the schema into a Cypher query (for use specifically with
```

Comment:
Unfortunately, the formatting rules about docstrings are pretty rigid on the headline always being a one-liner, so for the sake of not breaking with convention and tooling, let's try to play along.

How about: `Compile the GraphQL input into a RedisGraph Cypher query and associated metadata.` for the headline?

Reply:
Might be good to specify explicitly that it's only for RedisGraph?

`Compile the GraphQL input into a Cypher query (for RedisGraph only) and its metadata.` fits the line length rules. How's that sound? If it looks good, I'll update the other functions' headlines so they have a similar structure.

Reply:
Can you make this headline shorter?

Reply:
`Compile the GraphQL input into a Cypher query (for RedisGraph only) and its metadata.` does end up being one line.